### PR TITLE
Add rubygem-puma package

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
This is mainly to allow use in containerized deployments where we run Foreman with Puma.

File: packages/foreman/rubygem-puma/rubygem-puma.spec [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# FIXME:
```
Comment (ekohl) [inline packages/foreman/rubygem-puma/rubygem-puma.spec]: Could you remove these lines?

File: packages/foreman/rubygem-puma/rubygem-puma.spec [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name puma
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.11.4
+Release: 1%{?dist}
+Summary: Puma is a simple, fast, threaded, and highly concurrent HTTP 1.1 server for Ruby/Rack applications
+Group: Development/Languages
+License: BSD-3-Clause
```
Comment (ekohl) [inline packages/foreman/rubygem-puma/rubygem-puma.spec]: The Fedora guidelines prefer `BSD` according to https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses (and maybe we'll need a SPDX-to-Fedora mapping in gem2rpm).

**Comment by ehelms** [comment]
@ekohl Updated

**Pull request merged**